source/level/ecs/systems/physics/PhysicsSystem.h

```diff
@@ -69,14 +69,14 @@ class PhysicsSystem : public EntitySystem
      * Sometimes a move is possible but requires another move to be done as well,
      * therefore `moveToDo` will be changed to that other move (or Move::none)
      */
-    bool canDoMove(Physics &p, AABB &aabb, Move &moveToDo, Move prevMove);
+    bool canDoMove(Physics &physics, AABB &aabb, Move &moveToDo, Move prevMove);
```

Comment:
```suggestion
    bool canDoMove(Physics &, AABB &, Move &moveToDo, Move prevMove);
```
overbodige namen kan je trouwens zelfs weglaten in header files, is misschien overzichtelijker